docs/README.md

```diff
@@ -0,0 +1,624 @@
+Welcome to the *AIM API - Alpha*!
+
+# Alpha Notice
+
+The AIM API is in an "alpha stage" to gather customer feedback. While in alpha, the API may change in backwards incompatible ways to accommodate for fixes and additions. Breaking changes will be communicated to the primary API contact at least 5 business days in advance.
+
+TSG reserves the right to determine what constitutes a breaking changes. A definition of "breaking changes" will be made available before formal release.
+
+# HTTP API
+
+## API Discovery
+
+In order to reduce client coupling, the AIM API provides an API discovery document available at:
+
+<a class="discovery-config-url"></a>
```

Comment:
Al:
> - So you click on this link below and it is exactly what is below the link what is the purpose of this.
> - Is their actually a website for https://aim.thestrawgroup.com/api/auth/v1/ or https://aim.thestrawgroup.com/api/warehouse/v1 where one should expect to see documentation.

Reply:
I should go into a bit more detail about what this does. I added this at the top because it is referenced below in the "Obtain an ID Token" and "Query API" sections.